api-test/src/main/resources/pms/GetAllNotifications/GetAllNotifications.yml

```
@@ -989,4 +989,196 @@ GetAllNotifications:
+      "mispPartnerId": "$REMOVE$",
+      "mispLicenseKeyName": "$REMOVE$",
+      "notificationType": "weekly"
+}'
+      output: '{
+      "errors": [
+       {
+          "errorCode": "PMS_NOTIFICATION_ERROR_021"
+       }
+     ]
+}'
+   Pms_GetAllNotifications_with_Weekly_NotificationType_And_createdFromDate_And_createdToDate:
```

YAMLlint (1.38.0)

[error] duplication of key "Pms_GetAllNotifications_with_Weekly_NotificationType_And_createdFromDate_And_createdToDate" in mapping

(key-duplicates)